### PR TITLE
[fix] calibrate deployment

### PR DESCRIPTION
**Pull request by danielogen**
# [fix] calibrate deployment

**_WHAT DOES THIS PR DO?_**
This PR resolve issues with `calibrate` microservice suspected to originate from deployment 

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
N/A

**_WHAT IS THE LINK TO THE PR BRANCH_**
https://github.com/airqo-platform/AirQo-api/tree/fix-calibrate-deploy

**_HOW DO I TEST OUT THIS PR?_**
Test this PR using docker as follows:
Checkout this code locally, `cd` to `AirQo-api/src/calibrate`. Run `docker build -t calibrate:test -f Dockerfile.stage.txt .`. 

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
Once image creation is completed, run `docker run -it -p 4001:4001 calibrate:test`. Using a REST client of your choice, test the endpoint `[POST] http://localhost:4001/api/v1/calibrate`. Refer to the [API Documentation](https://docs.airqo.net/airqo-platform-api/calibrate) for details of the request body. A sample is provided below:
```
{
    "datetime": "2020-09-06 12:56:59+00:00",
    "raw_values": [
        {
            "raw_value": 100,
            "sensor_id": 832251
        },
        {
            "raw_value": 22,
            "sensor_id": 82720
        },
        {
            "raw_value": 29,
            "sensor_id": 870144
        }
    ]
}
```
**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
N/A

**_ARE THERE ANY RELATED PRs?_**
N/A

### CODE REVIEW

File: src/calibrate/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Inherit from the Python Docker image
 FROM python:3.7-slim
 
+RUN apt-get -y update
+RUN apt-get -y install git
+RUN apt-get -y install tk
```
Comment (ja-odur) [inline src/calibrate/Dockerfile:6]: Are we still using non-docker setup? I think this should be documented somewhere too (like in the readme) for cases where we are not using docker. Just thinking here 🤔 .

Comment (danielogen) [reply src/calibrate/Dockerfile:6]: Here, we are actually using Docker @ja-odur

**Review by Priscilla-AL — COMMENTED** [review]
C:\Users\User\Documents\Projects\AirQo-api\src\calibrate>docker run -it -p 4001:4001 calibrate:test.
Unable to find image 'calibrate:test.' locally
docker: Error response from daemon: pull access denied for calibrate, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.

**Comment by danielogen** [comment]
> C:\Users\User\Documents\Projects\AirQo-api\src\calibrate>docker run -it -p 4001:4001 calibrate:test.
> Unable to find image 'calibrate:test.' locally
> docker: Error response from daemon: pull access denied for calibrate, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.

Hi @Priscilla-AL , I can see a `.`(dot) at the end of you image name. Try removing it and test again. Thanks!

**Review by Priscilla-AL — APPROVED** [review]
looks good to me

**Review by ja-odur — APPROVED** [review]
LGTM

**Comment by danielogen** [comment]
This PR might longer be required. I will raise a separate PR to fix calibrate k8s service name

**Pull request closed without merging**